calculator.rb

```diff
@@ -0,0 +1,93 @@
+# Week#1: Calculator Build
+# Ana Lisa Sutherland - Octos C9
+
+# method for addition using two numbers
+def addition (num_one, num_two)
+  return num_one + num_two
+end
+#method for subtracting using two numbers
+def subtraction (num_one, num_two)
+  return num_one - num_two
+end
+#method for multiplication using two numbers
+def multiply (num_one, num_two)
+  return num_one * num_two
+end
+#method for division using two numbers
+def divide (num_one, num_two)
+  return num_one / num_two
+end
+#OPTIONAL: method for exponents
+def exponent (num_one, num_two)
+  return num_one ** num_two
+end
+#OPTIONAL: method for modulo
+def modulo (num_one, num_two)
+  return num_one % num_two
+end
+#OPTIONAL: ENSURE IT IS ONLY NUMBERS THAT ARE ENTERED
+def req ()
+  num = gets.chomp
+  until num =~ /^[1-9]\d*(\.\d+)?$/
```

Comment:
Good use of a regex here to verify user input - way to go above and beyond.

One comment: I'm not so sure about the method name `req`. This doesn't really tell me what the method does, which will make code that calls it harder to read. I would prefer something more descriptive like `read_number`.